### ISSUE TITLE
please add border background color

### ISSUE THREAD

**Issue by alz-ahm** [amulyakhare/TextDrawable#28]
Thanks for your great library. It would be great if you add a function to customize border background color as well.

**Comment by luca992** [comment]
yeah the ability to change the boarder color would be appreciated

**Comment by nat-e** [comment]
I created a PR for this: https://github.com/amulyakhare/TextDrawable/pull/34
